src/caches/beliefpropagationcache.jl

```diff
@@ -0,0 +1,221 @@
+default_mt_constructor(inds_e) = ITensor[denseblocks(delta(inds_e))]
```

Comment:
I think it is better to avoid using a shorthand like `mt` since I think it will be easy to forget what that means.

Maybe just `default_message_tensor`?

Reply:
Also, I realized until farther down that I had assumed `default_mt_constructor` would output a single message tensor. Maybe we could call this `default_message` to be more agnostic about the output type, i.e. it could in principle output a single `ITensor`, `Vector{ITensor}`, `ITensorNetwork`, etc. and those could all be considered messages.